articles/api/management/v1/use-cases.md

```diff
@@ -8,40 +8,24 @@ contentType: reference
 useCase: invoke-api
 ---
 
-# Management API v1 Use Cases
+# Features not affected by the Management API v1 Deprecation
 
-Currently, there are API features and functionality that are only available in the [Management API v1](/api/v1). If your business process or configuration requires these features, please continue to use the API v1. Otherwise, we recommend that you use the [new version](/api/v2) instead.
+If your business process or configuration requires these features, you can continue using these endpoints as they won't be affected by the Management API v1 Deprecation.
 
-The features only available in Management API v1 include:
+The features that are not being deprecated at the moment are:
 
 * [Active Directory Connector Monitoring](#active-directory-connector-monitoring)
-* [Application Users](#application-users)
-* [Email](#email)
 * [Enterprise Users/Directory Searching](#enterprise-users/directory-searching)
-* [Rules Configuration](#rules-configuration)
-* [Searching via the PSaaS Appliance](#searching-via-the-auth0-appliance)
 
 ## Active Directory Connector Monitoring
 
-In Management API v1, there is a `GET` endpoint that allows you to monitor the status of your Active Directory Connector:
+The following endpoint allows you to monitor the status of your Active Directory Connector. It is not currently available on the [Management API v2](/api/v2).
 
 GET `/api/connections/{AUTH0_CONNECTION}/socket`
```

Comment:
I think we have the exact same feature in APIv2: https://github.com/auth0/api2/blob/master/api/connections/routes/get_connection_status.js

Reply:
Yup! I added a JIRA task to track this thing: https://auth0team.atlassian.net/browse/CF-55

Reply:
socket wasn't public too, we can add this into migration docs regardless of adding it into APIv2 API docs.

Reply:
👍 Added!